.config/forest.dic

```diff
@@ -1,6 +1,11 @@
 10
 JWT
 JSON
+tipset/M
```

Comment:
what does `/M` mean?

Reply:
It means that it'll accept both `tipset` and `tipsets`.